README.md

```diff
@@ -7,8 +7,11 @@ The project is a multi-module Maven project. Each module contains an archetype f
 
 ### Prerequisites
 
-* JDK 11
 * Maven 3
+* Download GraalVM 19.3 or later
```

Comment:
shouldn't we point to their URL? https://www.graalvm.org/downloads/ (in case someone wants to choose enterprise).

Reply:
My personal preference is not to point users at a particular download location.